core/liblwm2m.h

```diff
@@ -667,6 +671,8 @@ typedef struct
     uint16_t                nextMID;
     lwm2m_transaction_t *   transactionList;
     void *                  userData;
+    int remaining_dataLen;
```

Comment:
To allow registration with multiple servers I think this needs to be in lwm2m_server_t instead.

Reply:
Okay.